Zend/zend.c

```diff
@@ -391,6 +391,8 @@ ZEND_API size_t zend_print_zval(zval *expr, int indent) /* {{{ */
 }
 /* }}} */
 
+extern ZEND_API inline size_t zend_print_variable(zval *var);
```

Comment:
No use of extern inline in php-src please.

Reply:
I must say I'm rather confused as to what is the difference between ``extern inline`` and ``static inline`` is, I used static inlines initially but when I talked to @morrisonlevi in R11 he preferred extern inline as apparently, IIRC, the semantics differ for ``static inline`` depending on the compiler. But I'm having a hard time finding relevant information about it.

Reply:
Yes, @morrisonlevi has opinions on this topic, but in php-src we do not use extern inline. It should be static inline or static zend_always_inline.

Reply:
Right, I will change them accordingly then.

Reply:
For what it is worth, it's not my _opinion_; it's actually the C99 and newer standard. The standards aren't freely available so it's hard to link, but here's GNU's explanation (with weird macros, but hopefully you can see through them): https://www.gnu.org/software/gnulib/manual/html_node/extern-inline.html.

Of course, compilers support `static inline` functions whether they are still standard or not, so I do acknowledge it is somewhat pedantic. The only reason I care is that you cannot use a `static inline` function inside of a plain `inline` function, which makes this non-standard use proliferate into my own codebase.